.travis.yml

```diff
@@ -8,10 +8,22 @@ python:
   - "3.6-dev" # 3.6 development branch
 
 # Install dependencies
-install: pip install flake8
+install:
+  - pip install -r requirements/base.txt
+  - pip install -r requirements/test.txt
+
+before_script:
+  - cd csunplugged/
+  - chmod +x manage.py
 
 # Runs test suite
-script: flake8
+script:
+  - flake8 --config=../.flake8
+  - coverage run --rcfile=../.coveragerc manage.py test --pattern "test_*.py" -v 3
+  - 'if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then ./manage.py test --pattern "test_*.py" --reverse -v 0; fi'
```

Comment:
is my interpretation correct: if the Travis' tests haven't failed up to this point, then run the test suite backwards